resources/js/features/homepage/components/findCityForm/CitySuggestions.tsx

```
@@ -29,7 +29,10 @@ export const CitySuggestions = ({
     return (
         <>
             {suggestions && (
-                <div className="w-full bg-white/80 text-black rounded-2xl overflow-hidden shadow-lg shadow-white/20 mb-4">
+                <div
+                    className="w-full bg-white/90 text-black rounded-2xl overflow-hidden shadow-lg shadow-white/20 mb-4 absolute top-14 left-0 z-10"
+                    role="listbox"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)